iCub/ros/rviz/iCubRviz.rviz

```diff
@@ -0,0 +1,733 @@
+Panels:
+  - Class: rviz/Displays
+    Help Height: 84
+    Name: Displays
+    Property Tree Widget:
+      Expanded: ~
+      Splitter Ratio: 0.5
+    Tree Height: 1043
+  - Class: rviz/Selection
+    Name: Selection
+  - Class: rviz/Tool Properties
```

Comment:
Is this panel required?

Reply:
The `.rviz` file is generate from rviz by saving the configuration. So, all the contents of the file are generated automatically and I did not manually edit the file.

Reply:
Deleting the panels sections opens rviz without any panels by default. Users can go to the panel drop down menu and add but I think it is unnecessary. So, I prefer to leave the panels section as such.

![Screenshot from 2019-06-04 09-55-49](https://user-images.githubusercontent.com/6505998/58861681-39e1aa00-86af-11e9-9e2d-8445936a23ca.png)

Reply:
`- Class: rviz/Tool Properties` section can be omitted safely. Rviz starts normally (i.e. without any changes in the panels) probably by loading default values for tools panel. Removed in https://github.com/robotology/icub-models/pull/23/commits/82b1b41444a8bdd1f29cd41fd25edce00fd23b62